wave_kinematics/src/rotation.cpp

```diff
@@ -0,0 +1,99 @@
+#include "wave/kinematics/rotation.hpp"
+#include "wave/utils/utils.hpp"
+
+double wave::Rotation::comparision_threshold = 1e-8;
```

Comment:
It's an initialization of a non-const static so you can't.. But it should not be a non-const static.